### PR TITLE
Refactored admin command

### PR DESCRIPTION
**Pull request by benwoo1110**

### CODE REVIEW

File: src/main/java/me/timmy109/betterreports/commands/AdminCommand.java [hunk 1]
```diff
@@ -42,53 +43,46 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
...
+					sender.sendMessage(Common.color("&cThere was an error reloading the config. Check console for more details."));
+				}
+				return true;
 		}
 
-		if (args.length > 1) {
-			sender.sendMessage(Common.color("&cUnknown Command"));
-			return true;
-		}
+		sender.sendMessage(Common.color("&cUnknown Command"));
```
Comment (mnewt00) [inline src/main/java/me/timmy109/betterreports/commands/AdminCommand.java]: There does not need to be an Unknown command message here, add a "default" statement to the switch block.

File: src/main/java/me/timmy109/betterreports/commands/AdminCommand.java [hunk 2]
```diff
@@ -42,53 +43,46 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
...
 
-			for (String s : playerHelp) {
-				sender.sendMessage(Common.color(s));
-			}
+			playerHelp.forEach(s -> sender.sendMessage(Common.color(s)));
+			return true;
+		}
+
+		if (args.length > 1) {
+			sender.sendMessage(Common.color("&cUnknown Command"));
```
Comment (mnewt00) [inline src/main/java/me/timmy109/betterreports/commands/AdminCommand.java:55]: Unknown command here is duplicated

File: src/main/java/me/timmy109/betterreports/commands/AdminCommand.java [hunk 3]
```diff
@@ -30,6 +30,7 @@
 import org.bukkit.command.CommandExecutor;
 import org.bukkit.command.CommandSender;
 import java.util.List;
+import java.util.stream.Stream;
```
Comment (mnewt00) [inline src/main/java/me/timmy109/betterreports/commands/AdminCommand.java]: Un-needed import, due to lambda statements being used.

**Review by mnewt00 — CHANGES_REQUESTED** [review]

File: src/main/java/me/timmy109/betterreports/commands/AdminCommand.java (hunk 2 above)
Comment (benwoo1110) [reply src/main/java/me/timmy109/betterreports/commands/AdminCommand.java:55]: Not really, unless you want users to run `/betterreports reload stuff` and the command is still valid?

Comment (mnewt00) [reply src/main/java/me/timmy109/betterreports/commands/AdminCommand.java:55]: See above, it's a duplicate of above.

**Review by mnewt00 — APPROVED** [review]
Code looks good, can be commented/explained but not-needed currently

**Pull request merged**